_data/sidebars/scos_dev_sidebar.yml

```diff
@@ -3886,6 +3872,8 @@ entries:
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-warehouse-data.html
           - title: HowTo - Import Packaging Units
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-packaging-units.html
+          - title: HowTo - Manage business unit-specific prices via data import
```

Comment:
Do we have `-` between `business unit-specific prices` or we just need a space there?